src/gdpr.ts

```diff
@@ -381,7 +381,7 @@
 //Telemetry.JupyterApiUsage
 /* __GDPR__
    "DATASCIENCE.JUPYTER_API_USAGE" : {
-     "extensionId": {"classification":"PublicNonPersonalData","purpose":"FeatureInsight","comment":"Extension Id that's attempting to use the API.","owner":"donjayamanne"},
+     "clientExtId": {"classification":"PublicNonPersonalData","purpose":"FeatureInsight","comment":"Extension Id that's attempting to use the API.","owner":"donjayamanne"},
```

Comment:
extensionId might be too generic as we're not getting any names, hence changing the property to somethign else.